run.py

```diff
@@ -73,15 +72,6 @@ def prepare():
     # place than it otherwise would be.
     os.environ['SASVIEW_DOC_PATH'] = joinpath(build_path, "doc")
 
-    # Make sure that we have a private version of mplconfig
-    #mplconfig = joinpath(abspath(dirname(__file__)), '.mplconfig')
-    #os.environ['MPLCONFIGDIR'] = mplconfig
-    #if not os.path.exists(mplconfig): os.mkdir(mplconfig)
-    #import matplotlib
-    # matplotlib.use('Agg')
-    # print matplotlib.__file__
-    #import pylab; pylab.hold(False)
-    # add periodictable to the path
```

Comment:
It was somewhat deliberate

Reply:
> Running python run.py after running python setup.py clean build install, I get an error that images in the sas/qtgui/images directory cannot be found. I haven't had a chance to trace this, but I don't see any time that directory was included with the build. This error is not present when running the main branch, though, so I'm not sure why it's happening now.

I think this is an issue with `setup.py`, this kind of thing is going to keep happening until its been shown some love. For now, it can probably just be added. Fixing the issues with `setup.py` is next on my list after this PR is accepted.

> Running the executable generated by the github workflow, the following error pops up and SasView fails to start. This might be a setuptools version issue and might not be exclusive to your branch, but I haven't tested other recently built executables.

Ah yeah, that's because of StrictVersion - I don't think it's really needed if we do sensible versioning. Should be an easy fix.

Reply:
Not really accidental.